scene/main/viewport.cpp

```diff
@@ -2607,6 +2633,7 @@ void Viewport::_drop_mouse_focus() {
 void Viewport::_drop_physics_mouseover(bool p_paused_only) {
 	physics_has_last_mousepos = false;
 
+	// TODO: deduplicate this with the erasing in ss2d.
```

Comment:
What does SS2D mean? I'd replace it with the full expanded text.

Reply:
ss2d is an object in this file. I added better pointers (but still want to point where in the method the place I point to is, as the method is quite long).